modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/UserAccountResourceDTOConverter.java

```diff
@@ -231,9 +237,10 @@ public UserAccount toDTO(DTOConverterContext dtoConverterContext, User user)
 		};
 	}
 
-	private <T, E extends Exception> T _getContactField(
-			Contact contact, UnsafeFunction<Contact, T, E> fieldFunction)
-		throws E {
+	private <T> T _getContactField(
```

Comment:
You can remove this function now, don't you?

Reply:
What I removed is the use of the lambda expression because is the code that Brian highlighted. The function is still used [here](https://github.com/liferay-frontend/liferay-portal/blob/ba9eab1ab845936057fd506b19fcb9e473ac98c4/modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/UserAccountResourceDTOConverter.java#L152-L164)
With this I avoid a lot of boilerplate code, I hope Brian will not find ambiguous those pieces of code too.